desktop/apps/auction/test/routes.js

```diff
@@ -102,7 +102,7 @@ describe('#redirectLive', () => {
     RoutesRewireApi.__Rewire__('metaphysics', sinon.stub().returns(Promise.resolve(auctionQueries)))
     res = {
       redirect: (url) => {
-        url.should.equal('https://live.artsy.net/foo/login')
+        url.should.equal(`${process.env.PREDICTION_URL}/foo/login`)
```

Comment:
I'm not sure `process.env` is available here, and in general it shouldn't be used (these things should all go through `sharify`)